include/hipSYCL/sycl/libkernel/accessor.hpp

```diff
@@ -1221,63 +1228,125 @@ using unranged_placeholder_accessor =
 // Accessor deduction guides
 #ifdef HIPSYCL_EXT_ACCESSOR_VARIANT_DEDUCTION
  #define HIPSYCL_ACCESSOR_VARIANT_SELECTOR(TagT, fallback, optimized) \
-  detail::deduce_accessor_variant<TagT>(optimized)
+  detail::deduce_accessor_variant(TagT, optimized)
```

Comment:
`TagT` implies a type, bug as below you rather would want `tag`, as it's a value, I'd say.

Reply:
Thanks, fixed :)